WikilogUtils.php

```diff
@@ -365,7 +365,7 @@ public static function tagList( $list ) {
 	public static function splitSummaryContent( $parserOutput ) {
 		global $wgUseTidy;
 
-		$content = Sanitizer::removeHTMLcomments( $parserOutput->getText() );
+		$content = Sanitizer::removeHTMLcomments( $parserOutput->getContent()->getNativeData() );
```

Comment:
same as above.

Reply:
I think `$title` here is not related with `WikiPage`